W-11640220-permissions-req-change (#241)
mulesoft/docs-monitoring: modules/ROOT/pages/am-permissions.adoc

include::_attributes.adoc[]
endif::[]

> Anypoint Monitoring has two product-specific permissions that control access to assets within Anypoint Monitoring:

Reviewer: Suggestion to consider:  Anypoint Monitoring has two product-specific permissions that control access to its assets:

Author: ```suggestion
Anypoint Monitoring has two product-specific permissions that control access to its assets:
```

Anypoint Monitoring Administrator:: Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
Anypoint Monitoring Viewer:: Allows users to view but not modify content in Anypoint Monitoring.

The content you can access when you have the Anypoint Monitoring Administrator and Anypoint Monitoring Viewer permissions depends on other asset-specific permissions.
To view or modify content related to specific apps and APIs, you also need the appropriate access permissions for those assets.

> For example, a user who has the Anypoint Monitoring Viewer permission who wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment as the app.

Reviewer 2: "Read Applications" -> "Runtime Manager Read Applications"? 
Would it be a good idea to keep the pattern `{product namespace} {permission name}`?

Reviewer 3: We should explicitly specify the permissions that are required for monitoring data e.g. read applications.

Author: I hesitate to do that since these permissions models may change. There are also lots of variables for which permissions are needed based on what the users need to see.

Author: ```suggestion
For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment that the app is deployed to.
```

> For example, a user who has the Anypoint Monitoring Viewer permission who wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment as the app.

Reviewer: ```suggestion
For example, a user who has the Anypoint Monitoring Viewer permission and who wants to view Anypoint Monitoring data for an on-premises app also needs the `Read Applications` permission in the same environment as the app.
```

Reviewer: I put Read Applications in back ticks but don't know if that is consistent with the rest of the doc.

Author: We do not put permissions in backticks for now.

> For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to.

Author: ```suggestion
For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to.

At minimum, you need the following permissions to read data in Anypoint Monitoring:

* View API Alerts (API Manager) for Flex API content
* Read Applications (Runtime Manager) for on-premises app content
* Read Servers (Runtime Manager) for Flex Gateway content
```

== See Also

* xref:access-management::teams.adoc[Access Management Teams]
* xref:access-management::roles.adoc[Access Management Roles]
* xref:access-management::managing-permissions.adoc[Access Management Managing Permissions]
* xref:access-management::external-organization-access.adoc[Access Management External Organization Access]
* xref:api-manager::environment-permission-task.adoc[Permissions in API Manager]

> * xref:cloudhub-2::ps-assign-permissions.adoc[Permissions in CloudHub 2.0]

Reviewer 2: Do we have API Manager permissions docs?

Author: I am finding that we don't have proper permissions docs in API Manager, CH1, RTF, or RTM. 😬😬 I am working on this from the IAM side.
